recipes/astrometry/meta.yaml

```
@@ -0,0 +1,68 @@
+{% set name = "astrometry" %}
```

[Periods should be fine](https://docs.conda.io/projects/conda-build/en/latest/concepts/package-naming-conv.html#index-0) and conda-forge already has a lot of packages that use them:
```suggestion
{% set name = "astrometry.net" %}
```

---

I know dots are allowed, but there is something that creates a buggy version of the package if I include the `.net`. Look these two Travis-CI results [travis-1](https://travis-ci.com/juliotux/astropop/jobs/232078269) and [https://travis-ci.com/juliotux/astropop/jobs/232158172](travis-2). The first one is using `astrometry.net` as dependency and the second one `astrometry` as dependency. The first one is failing due to the buggy version of `astrometry.net` package, getting this error:

```
CondaValueError: Malformed version string '.NET_VERSION': empty version component.

Installing the dependency astrometry.net with conda was unsuccessful, using pip instead.

ERROR: Invalid requirement: 'astrometry.net.NET_VERSION.*'
```

This is present even for the most recent stable versions of conda like in [travis-3](https://travis-ci.com/juliotux/astropop/jobs/232152875) CI build.

---

Also, the program has a python `setup.py` file (that just calls the make process) that names this package as `astrometry` and not `astrometry.net`. So, `astrometry` can keep more consistency.